Lib/queue.py

```diff
@@ -84,10 +100,17 @@ def join(self):
         to indicate the item was retrieved and all work on it is complete.
 
         When the count of unfinished tasks drops to zero, join() unblocks.
+
+        Raises a ShutDown if shutdown_state attribute is set to
+        `_queue_shutdown_immediate`.
```

Comment:
```suggestion
        Raises ShutDown if the queue has been shut down immediately.
```

Doc-strings are for humans